.github/workflows/extension_upgrade.yml

```diff
@@ -22,7 +22,7 @@ on:
 jobs:
   test:
     name: Upgrade Test
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
updated this so that I can install `postgresql-server-dev-16` which is apparently not available on 22.04